crates/ruma-events/src/room/message/feedback.rs

```diff
@@ -1,4 +1,6 @@
-//! Types for the `m.room.message.feedback` event.
+//! Types for the [`m.room.message.feedback`] event.
+//!
+//! [`m.room.message.feedback`]: https://spec.matrix.org/v1.1/client-server-api/#events-19
```

Comment:
This link is incorrect. Like `m.room.aliases`, this is an event that is no longer used (for a long time, maybe we should just remove it altogether).